### PR TITLE
Fixing highlight issue

### PR DESCRIPTION
**Pull request by atharva-malik**
There is a slight issue with the website, nothing more than a personal ick, but if you highlight the pink sections, they "disappear" I fixed that in this PR.

image
image

### CODE REVIEW

File: src/components/Locations.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default function Locations() {
     "San Francisco, CA",
     "Burlington, VT",
     "Oshkosh, WI",
+    "Atlanta, GA"
```
Comment (samdev-7) [inline src/components/Locations.js:17]: This is unrelated to the PR. Please remove it and move it to another PR. Thanks!

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by atharva-malik** [comment]
Hey @samdev-7, should be fixed, check again thx

**Review by samdev-7 — APPROVED** [review]
lgtm, ty!
